codestodians/food-basket-front-end
# [Bug: #163762582] Fix profile page render error on Login

#### What does this PR do?
- Fixes the render error that breaks the app on user login

#### Description of Task to be completed?
- Retrieve response on login, save useful details to local storage

#### How should this be manually tested?
- Check out into the `bug/get-user-details-on-login` branch.
- Start both servers
- Navigate to the `/profile` route.
- The profile page should render successfully.

#### Any background context you want to provide?
- The userId is vital for retrieving the user details, so has to be included in the login mutation as it was in the signup mutation

#### What are the relevant pivotal tracker stories?
- [#163762582](https://www.pivotaltracker.com/story/show/163762582)
